### PR TITLE
Feat: Add workflow

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Add a workflow that updates the SNS Aggregator files for the ProdLaunchpad.svelte test.

# Changes

* New workflow `update-sns-aggregator-response` that runs every thursday.

# Tests

Example of the PR that will create: https://github.com/dfinity/nns-dapp/pull/4339

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4337?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #4336** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4336?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #4337** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4337?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4337?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd @bitdivine please review

I believe this is my first workflow I add alone 🎊

**Comment by dskloetd** [comment]
This is the second PR I see without PR description. Did something break in Graphite?

File: .github/workflows/update-sns-aggregator-response.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    branches:
+      # Run when the development branch for this workflow is updated.
+      - update-aggregator-response
+jobs:
+  update-aggregator-response:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Update the SNS aggregator response files
+        id: update
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply .github/workflows/update-sns-aggregator-response.yml]: thanks, removed

**Comment by lmuntaner** [comment]
@dskloetd ready for another review

**Review by dskloetd — APPROVED** [review]

**Pull request merged**